### PR TITLE
290 replace contents in metadata with content

### PR DESCRIPTION
**Pull request by tcnichol**
Testing some extractors I realized the field that was 'content' in v1 was changed to 'contents' in v2, which caused a number of extractors not to work with clowder v2. 

With this pull request I have changed 'contents' to 'content' everywhere, in the back end and front end, hoping that this will reduce the amount of changes that might need to be made to older extractors or in pyclowder. 

Currently wordcount extractors works correctly. 

Other extractor related issues will be taken care of in other pull requests.

### CODE REVIEW

**Review by longshuicy — APPROVED** [review]
Pytest is failing again? Besides that looks good. Approved.

**Comment by tcnichol** [comment]
Fixing pytest, should be done and pushed soon.

File: frontend/src/components/files/UploadFile.tsx [hunk 1]
```diff
@@ -48,8 +48,9 @@ export const UploadFile:React.FC<UploadFileProps> = (props: UploadFileProps) =>
 		setMetadataRequestForms(prevState => {
 			// merge the contents field; e.g. lat lon
 			if (metadata.definition in prevState){
-				const prevContent = prevState[metadata.definition].contents;
-				metadata.contents = {...prevContent, ...metadata.contents};
+				const prevContent = prevState[metadata.definition].content;
+				metadata.content = {...prevContent, ...metadata.content};
+				metadata.content = {...prevContent, ...metadata.content};
```
Comment (max-zilla) [inline frontend/src/components/files/UploadFile.tsx:53]: looks like a duplicate line here

**Review by max-zilla — CHANGES_REQUESTED** [review]
aside from one line that looked duplicated, changes look fine and i didnt notice any issues running things. will approve & merge once removed

**Comment by tcnichol** [comment]
I merged main into this one and need to check and make sure it still works.

**Review by max-zilla — APPROVED** [review]

**Pull request merged**